backend/labels/views.py

```diff
@@ -118,6 +118,14 @@ class ScaleListAPI(BaseListAPI):
     label_class = Scale
     serializer_class = ScaleSerializer
 
+    def create(self, request, *args, **kwargs):
+        if self.project.is_affective_annotation_project:
+            queryset = Scale.objects.filter(example=self.kwargs["example_id"], label=self.kwargs['label_id'])
+            print(queryset)
+            if not self.project.collaborative_annotation:
+                queryset = queryset.filter(user=self.request.user)
+            queryset.delete()
+        return super().create(request, args, kwargs)
```

Comment:
Thanks for the quick fix! Unfortunately I have tried it and I got a key error from self.kwargs['label_id'].

When I tried to print kwargs, it only contains project id and user id. Meanwhile `request.data` contains something like this: `{'id': 0, 'label': 24, 'scale': 8, 'user': 0}` (The 'label' field contains the right label_id). So I think we need to adjust the code with something like: `label=request.data['label']`

Reply:
I have tested it once more and I can confirm changing from `label=self.kwargs['label_id']` to `label=request.data['label']` fixes the issue :)

Reply:
ah yeah thank you for pointing it XD I forgot to check it XD updated